cypress/integration/Crosscutting/PageOptionsTest/index.js

```diff
@@ -0,0 +1,65 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+//This step is just a workaround until it gets fixed
```

Comment:
Can you please create a ticket to remove this step once this error is resolved in the digital-platform? ( provide a link to the platform ticket for reference)

Reply:
Done. Entered (#118 )